src/Components/Subscriptions/Table/index.jsx

```diff
@@ -136,12 +143,20 @@ const Table = ({ data }) => {
         </div>
       )}
       {showConfirmDeleteModal && (
-        <ConfirmDeleteModal
-          onClose={() => setShowConfirmDeleteModal(false)}
-          confirmDelete={() => handleDelete(editingSubscriptionId)}
-        />
+        <ConfirmModal
+          title="Delete Subscription"
+          handler={closeModal}
+          onAction={() => handleDelete(editingSubscriptionId)}
+          reason="delete"
+        >
+          <p>Are you sure to delete subscription?</p>
+        </ConfirmModal>
+      )}
+      {showDeleteModal ? (
+        <ResponseModal handler={closeModal} state="fail" message="Subscription Deleted" />
```

Comment:
It's not a good practice to pass an empty div. It would be better to use something like this: "createModal &&"
(I did it like this last week and had a comment in my feedback)